lib/bundle-validator.js

```diff
@@ -408,31 +411,100 @@ class BundleValidator {
         const assetsPath = path.join(this.themePath, 'assets');
         const stylesPath = path.join(this.themePath, 'assets/scss');
         const rawConfig = await this.themeConfig.getRawConfig();
-        const compiler = rawConfig.css_compiler;
-
-        let cssFiles;
-        try {
-            const files = await fs.promises.readdir(stylesPath);
-            cssFiles = files.filter((file) => {
-                return file.slice(-(compiler.length + 1)) === `.${compiler}`;
-            });
-        } catch (e) {
-            throw new Error(`Error: ${e.message}, while reading files from "${stylesPath}".`.red);
-        }
+        const cssFiles = await this.getCssFiles();
 
         for (const file of cssFiles) {
             try {
+                const engine = stencilCLISettings.isOldNodeSassForkEnabled()
+                    ? cssCompiler.FALLBACK_SASS_ENGINE_NAME
+                    : cssCompiler.SASS_ENGINE_NAME;
                 /* eslint-disable-next-line no-await-in-loop */
-                await cssCompiler.compile(rawConfig, assetsPath, file);
+                await cssCompiler.compile(rawConfig, assetsPath, file, engine);
             } catch (e) {
                 const message = this.parseStencilStylesError(e);
+                if (!stencilCLISettings.isOldNodeSassForkEnabled()) {
+                    throw new Error(
+                        `${message}\n`.red +
+                            `\n` +
+                            `---------WARNING---------\n`.red +
```

Comment:
Are you okay with such wordings or do we need to adjust?

cc @bookernath

Reply:
Looks good 👍  is there some link we can provide to help people diagnose errors? Some node-sass documentation perhaps?

Reply:
There are bunch of links, that you need to really dig to understand the context:
https://github.com/sass/node-sass/releases/tag/v3.5.1 - the release, that broke and changelogs.
https://github.com/bigcommerce/cornerstone/pull/999/files#diff-7b10e2a8164a3ef7ccb5aa035f393c9715b8945fcc59613d455408c505dd42a5L4 - known example of broken theme case (cornernstone)
https://github.com/sass/libsass/issues/1550 - code examples
https://github.com/sass/sass/issues/779 - code examples

@bookernath Maybe we can include all of them into out changelog and show the link for the changelog?